platform/search/bundle.json

```diff
@@ -0,0 +1,33 @@
+{
+    "name": "Search",
+    "description": "Allows the user to search through the file tree.",
+    "extensions": {
+        "components": [
+            {
+                "provides": "searchService",
+                "type": "provider",
+                "implementation": "providers/GenericSearchProvider.js",
+                "depends": [ "$q", "objectService", "workerService", "roots[]" ]
+            },
+            {
+                "provides": "searchService",
```

Comment:
I would move this particular `searchService` into platform/persistence/elastic, so that it gets exposed along with the ElasticSearch persistence provider